src/themes/amazon/elements/button.overrides

```diff
@@ -1,5 +1,5 @@
 .ui.button {
-    background-image: linear-gradient(center top, #f7f8fa, #e7e9ec) repeat scroll 0 0 rgba(0, 0, 0, 0);
+    background-image: linear-gradient(to top, #f7f8fa, #e7e9ec) repeat scroll 0 0 rgba(0, 0, 0, 0);
```

Comment:
`center` seems to be really not valid, how can it be fixed?

Reply:
The whole line is invalid (and even the gradient direction is wrong (i tested the theme)
a valid value is
```suggestion
    background: linear-gradient(to bottom, #f7f8fa, #e7e9ec) repeat scroll 0 0 rgba(0, 0, 0, 0);
```